admin/services/data-access/check-window.data.service.js

```diff
@@ -63,7 +64,7 @@ const checkWindowDataService = {
         type: TYPES.DateTimeOffset
       }
     ]
-    const sql = 'UPDATE [mtc_admin].[checkWindow] SET isDeleted=1, updatedAt=@updatedAt WHERE id=@id'
+    const sql = `UPDATE ${sqlService.adminSchema}.${table} SET isDeleted=1, updatedAt=@updatedAt WHERE id=@id`
```

Comment:
not sure if you are up to date with `data-layer-development` branch but there *may* be tweaks to some of these already

Reply:
It was up to date yesterday lunch time. Will merge latest in again to bring it bang up to date before merge.

Reply:
👍